web/src/main/webResources/WEB-INF/web.xml

```diff
@@ -429,6 +429,22 @@
   </servlet-mapping>
 
 
+  <servlet>
+    <servlet-name>jolokia-agent</servlet-name>
+    <init-param>
+      <param-name>policyLocation</param-name>
+      <param-value>classpath:/jolokia-access.xml</param-value>
+    </init-param>
+    <servlet-class>org.jolokia.http.AgentServlet</servlet-class>
```

Comment:
![image](https://user-images.githubusercontent.com/1701393/51405142-d585da00-1b55-11e9-9787-ccd7a33961f5.png)

Is this something we should fix ?